lms/djangoapps/teams/static/teams/js/views/team_profile.js

```diff
@@ -78,9 +78,9 @@
                     event.preventDefault();
                     var view = this;
                     ViewUtils.confirmThenRunOperation(
-                        gettext("Leave this team?"),
-                        gettext("If you leave, you can no longer post in this team's discussions. Your place will be available to another learner."),
-                        gettext("Confirm"),
+                        _.escape(gettext("Leave this team?")),
```

Comment:
@symbolist all these `_.escape` are needed here because template is not doing escape.